lib/vimgolf/cli.rb

```diff
@@ -2,7 +2,6 @@ module VimGolf
 
   GOLFDEBUG    = ENV['GOLFDEBUG'].to_sym rescue false
   GOLFHOST     = ENV['GOLFHOST']     || "http://www.vimgolf.com"
-  GOLFDIFF     = ENV['GOLFDIFF']     || 'diff'
```

Comment:
By "diff viewers" I really think you mean `GOLFSHOWDIFF`. The function of `GOLFDIFF` was to name an external `diff` command to tell you if your solution is right. I replaced that with the `challenge.correct?` method, so `GOLFDIFF` isn't needed any more.